docs/team/chewterence.md

```diff
@@ -12,30 +12,63 @@ regularly on the specified days and times every week.
-  * Wrote additional tests for updated Add and Edit commands to increase coverage for the new fields (Pull requests [\#56](https://github.com/AY2021S2-CS2103T-W10-1/tp/commit/85b7432bce0d7bbc5dba1b961959e919b297f6d8))
+* **Enhancements to existing features**:
+  * Updated `add` and `edit` commands to utilize the new fields added (Pull requests [\#52](https://github.com/AY2021S2-CS2103T-W10-1/tp/commit/cc6891e016c15be52a00996aa8a74f383efb2e7e))
+    * What it does: The enhancement allows users to add and edit passengers with the new fields of TripDay, TripTime and Price.
+    * Justification: TripDay, TripTime and Price fields were added to the model and thus the `add` and `edit` commands were required to
+      be updated with the new fields.
+    * Highlights: A great amount of effort was put into ensuring that the implementation of the code was of suitable quality. This included
+      writing additional tests to increase coverage for methods that involved the new fields 
+      (Pull requests [\#56](https://github.com/AY2021S2-CS2103T-W10-1/tp/commit/85b7432bce0d7bbc5dba1b961959e919b297f6d8)).
+      Moreover, the addition of the Price field proved to be far more challenging than expected, as it was an optional field that the user could enter.
+      The Price field was thus implemented in a way that avoided the use of nulls as much as possible, thereby wrapping it in a java Optional.
 
 * **Documentation**:
   * User Guide:
-    * Added user guide introduction and quick start
-    * Added documentation for the command summary and FAQ
-    * Updated documentation for Add and Edit commands
-    * Added documentation for Pool and Unpool commands
-
-* **Community**:
-  * PRs reviewed (with non-trivial review comments): [\#57](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/57)
-  * Contributed to forum discussions (examples: [Streams VS Loops Discussion](https://github.com/nus-cs2103-AY2021S2/forum/issues/145))
-  * Reported bugs and suggestions for other teams in the class (examples: )
+    * Drafted initial version of the User Guide, added user guide introduction and quick start.
+    * Added documentation for the command summary and FAQ.
+    * Updated documentation for `add` and `edit` commands to match newly added fields of TripDay and TripTime.
+    * Added documentation for `pool` and `unpool` commands.
+  * Developer Guide:
+    * Added documentation for `pool` and `unpool` commands (Pull requests [\#234](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/234)).
+    * Added UML sequence and activity diagrams for `pool` and `unpool` commands (Pull requests [\#234](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/234)).
+    * Added instructions for manual testing documentation for `pool` and `unpool` commands (Pull requests [\#277](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/277)).
+    * Added Effort Appendix page (Pull requests [\#280](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/280)).
```

Comment:
minor thing but maybe section instead of page?

Reply:
thanks for the suggestion will make the changes